types/jsdom/base.d.ts

```diff
@@ -2,7 +2,7 @@
 /// <reference types="node" />
 
 import { EventEmitter } from "events";
-import { ElementLocation } from "parse5";
+import { Location } from "parse5/dist/common/token";
```

Comment:
```suggestion
import { Token } from "parse5";
```

then `Token.ElementLocation`

Reply:
ah fair, I just tested in TS playground, which doesn't care about `@internal`

Reply:
also note that maintainers says to use `Token` export, so probably an error on their side: https://github.com/inikulin/parse5/issues/555#issuecomment-1137806642

Reply:
for now i have opened https://github.com/inikulin/parse5/issues/624

Reply:
changed it back in df0d91e0dd5202135ec9bbda5a7593e658878e23 according to response https://github.com/inikulin/parse5/issues/624#issuecomment-1193088497